packages/cloud-cognitive/src/components/Datagrid/Datagrid/addons/CustomizeColumns/Columns.js

```diff
@@ -31,13 +35,30 @@ const getNextIndex = (array, currentIndex, key) => {
 };
 
 const Columns = ({
+  setVisibleColumns,
+  getVisibleColumnsCount,
```

Comment:
commented out code should really be cleaned out (there are of course exceptions).

Reply:
This code is not commented out and therefore should not be removed.